Packages/com.chisel.editor/Chisel/Editor/Editor/ChiselUnityEventsManager.cs

```diff
@@ -56,11 +56,11 @@ public static void Initialize()
             UnityEditor.Undo.postprocessModifications					+= OnPostprocessModifications;
 
 #if UNITY_2019_1_OR_NEWER
-            UnityEditor.SceneView.duringSceneGui					    -= OnSceneGUI;
-            UnityEditor.SceneView.duringSceneGui                        += OnSceneGUI;
+            UnityEditor.SceneView.beforeSceneGui                        -= OnSceneGUI;
```

Comment:
Unrelated to this PR specifically, but could we get a "Why" comment at the top of this function to explain removing the event listener before adding it